xbmc/windowing/XkbcommonKeymap.cpp

```diff
@@ -0,0 +1,351 @@
+/*
```

Comment:
It really should be used by mir, too, that's why I put it into a common folder. I can move the file or commit the mir change.